indico/modules/events/registration/client/js/form/fields/MultiChoiceInput.jsx

```diff
@@ -188,6 +189,16 @@ export default function MultiChoiceInput({
       placesUsed={placesUsed}
       existingValue={existingValue}
       isEqual={_.isEqual}
+      validate={value => {
+        if (maxChoices && _.keys(value).length > maxChoices) {
+          return PluralTranslate.string(
+            'At most one option can be selected',
```

Comment:
```suggestion
            'At most {n} option can be selected',
```

iirc this works better for translations in languages with no plural forms